info.json

```diff
@@ -52,13 +64,23 @@
                 "name": "Excludelist_Domains",
                 "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com,fortiguard.com,forticloud.com,fortielab.com",
                 "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com,fortiguard.com,forticloud.com"
+            },
+            {
+                "id": 33,
+                "name": "Excludelist_Emails",
+                "value": "ena.vuk@bhtelecom.ba,sysadmin@bhtelecom.ba",
```

Comment:
@cs-ankit-gupta Any specific reason, why these emails are used in Excludelist_Emails?

Reply:
These were the default values